src/components/RegisterPage/RegisterForm.tsx

```diff
@@ -227,12 +228,20 @@ export const RegisterForm = ({ onSuccess }: RegisterFormProps) => {
         t={t}
       />
       <Consent error={errors.consents} onChangeFn={handleConsentsChange} />
+      <ReCAPTCHA
+        ref={recaptchaRef}
+        size='invisible'
+        badge='bottomright'
+        sitekey='6Ld0A54qAAAAANJ8mLCpJAxEp0XKtJyueFmEFVaG'
```

Comment:
It would be the best to extract this to the `.env` file. in the `.env` file, please create a new variable called something like `VITE_RECAPTCHA_SITE_KEY`. then, here you can use it as follows:


```suggestion
        sitekey=import.meta.env.VITE_RECAPTCHA_SITE_KEY
```